package.json

```diff
@@ -91,7 +91,6 @@
     "react-docgen": "^2.19.0",
     "react-dom": "^16.7.0",
     "react-router": "3",
-    "react-scroll": "^1.7.7",
```

Comment:
Did you confirm no other packages used this?

Reply:
In the screenshot you can see that it is only found in the `formation-react` package.

Reply:
Ahh, I totally overlooked the screenshot. :see_no_evil: Good call. :+1: